### DISCUSSION TITLE
[CONTRACTS] - Competition vs uAD

### DISCUSSION THREAD

**Discussion by 0x4007** [pavlovcik/uad-ui#138] [Q&A] [answered]
Compelling pitches for why uAD over the following:

- [x] FRAX vs uAD
- [x] UST vs uAD
- [x] FEI vs uAD

@hpiyankov

**Comment by hpiyankov** [comment, marked as answer]
All of the below are comments based on very limited research - about 1-2 hours per stable coin.  If you need me I can do more in-depth modelling, but it sounds to me like we are short on time.

1. FRAX seems like a very well tough out and currently functioning system. Our main advantages compared to them are:
- We are a lot more capital efficient (their current collaterisaiton ratio is 82% and this is the lowest it has been for some time now). 
- The price of FXS seems to be totally uncorrelated with the price of FRAX which is hood for stability, but it is questionable if it is actually profitable to stabilize the protocol in the long run.

In a nutshell their system is unproven with very low CRatios and currently is about the equivalent of minting a stablecoin where you give on DAI and you get 1 FRAX in return. It is stable by default, but not at all capital efficient or profitable.

2. UST - Terra has similar stabilisation mechanisms as us, but they are less sophisticated. They have one major advantage which offsets this - they are a native stable coin to a standalone blockchain that can afford to manipulate their core currency in order to stabilize the coin. The equivalent of this would be the Ethereum Foundation to issue a stable coin and pledge to stabilize it via mined ETH - we cannot compete with that.

The core topic here is - are you an Ethereum maxi? If you are, then the Terra blockchain will go away after some time, since it will not be competitive with Ethereum, and then UST will collapse. 

So our core advantages here are: we are not pegged to the success of the Terra blockchain which is still in its infancy. 

Another topic is that they seem to have had a couple of quite significant de-pegging events, however, they managed to bounce back from them.

3. FEI - at this point in time I really don't know what is going on here. In a nutshell - FEI is a stable coin whose main selling point is direct incentives....but the direct incentives are disabled. Apart from this, their only other incentivisation mechanic is arbitrage. 

The protocol's market cap just dropped in August from 2bn to 500mm where it has been since then. 

They have 1 bn protocol controlled value and 500 MM tokens in circulation, out of which 200k are controlled by the protocol. Essentially they are 300% colateralised. This is way worse capital efficiency than even DAI and it is better only to the Synthetix USD.

**Reply by 0x4007** [reply]
> but it is questionable if it is actually profitable to stabilize the protocol in the long run.

Can you elaborate on this I'm not following

**Reply by hpiyankov** [reply]
The main profit for stabilisation of the protocol comes from the FXS shares. They are not seigniorage shares, they are a utility token that is required in order to redeem FRAX. Those are single-use, eg. there is no continuous cash flow, like in our case. There is buyback & burn though.

Right now, I am not seeing a clear pattern between FXS appreciation and the FRAX cycles, which means that I am not 100% sure how and if people are actually profiting from keeping the peg.
